docs/source/design/load.rst

```diff
@@ -2,4 +2,13 @@
 Load Stage
 ==========
 
-- Target API config
+The Load stage feeds record data through a plugin interface that must be
+implemented for each target service.
+
+Target service plugins implement the necessary routines for converting record
+data coming in from the previous stage into message payloads and also
+implement the logic for sending those payloads to the target service.
+
+.. image:: /_static/images/load_stage.svg
```

Comment:
I think this is a really nice image which illustrates how the loader works but there are some key concepts that are a bit vague probably need some explanation. Can you add:

1. Something that describes what the unique key is, how its used, what the build key function is
2. Something that shows an example target entity (maybe even add an image of a JSON entity to your graphic for completeness)

Also, can you add the following examples (probably in How-To section):

1. Using the target API ingest plugin with the ingest pipeline (adding it to app settings, running `kidsfirst` with the custom app settings, etc.)

2.  Using the load stage as a standalone loader with a data file. 
     Basically exactly how we use it in the integration test? I'm thinking this is how most non-Kids First users will 
     use it (specifically thinking of NCPI FHIR WG participants who have their own data in csv/tsv files 
     somewhere).

Reply:
> Using the load stage as a standalone loader with a data file.

I'm not sure exactly what you mean by this, but it sounds like something that I'd like to cover in a different PR after talking about it.

Reply:
done